columnflow/plotting/plot_util.py

```diff
@@ -55,6 +57,97 @@ def get_cms_label(ax: plt.Axes, llabel: str) -> dict:
     return cms_label_kwargs
 
 
+def round_dynamic(value: int | float) -> int | float:
+    """
+    Rounds a *value* at various scales to a subjective, sensible precision. Rounding rules:
+
+        - 0 -> 0
+        - (0, 1) -> round to 0.1
+        - [1, 20]: round to 1
+        - (20, 100]: round to 5
+        - (100, 200]: round to 10
+        - (200, 500]: round to 20
+        - (500, 1000]: round to 50
```

Comment:
I thought about that (for symmetry reasons :) ), but rounding to 2 is never used anywhere. But then I agree, we should maybe remove the rounding to 20 as well.

Reply:
I'd personally prefer to keep the step, but since this type of rounding is only used for plotting parameters and not to round any physical quantities, I'm fine with it.